lms/services/moodle.py

```diff
@@ -138,6 +138,8 @@ def list_pages(self, course_id: int):
     def _get_contents(contents, parent=None):
         file_paths = []
         for content in contents:
+            if content["mimetype"] != "application/pdf":
```

Comment:
Do we know how does Moodle infer media types? Can this value be trusted?

Reply:
Not entirely sure. In other LMS (blackboard?) we just do endwith("pdf") so this is probably more reliable.